internal/common/windows_utils.bzl

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def is_windows(ctx):
+    return ctx.configuration.host_path_separator == ';'
```

Comment:
This is an ugly hack, the reason is that we don't have a proper way to tell the target platform in Starlark rule implementation.

There is another way to distinguish the platform,
https://github.com/bazelbuild/bazel-skylib/blob/master/rules/diff_test.bzl#L143
Maybe it's a litter better.